src/WORKSPACE

```diff
@@ -73,8 +73,8 @@ boost_deps()
 new_git_repository(
     name = "eigen",
     build_file = "@//external:eigen.BUILD",
-    commit = "21ae2afd4edaa1b69782c67a54182d34efe43f9c",
-    remote = "https://gitlab.com/libeigen/eigen.git",
+    commit = "2bf07fa5b5034ed5f5b1839fea40a2581946f6fa",
```

Comment:
Changing this commit might be causing the simulated tests to fail, @akhilveeraghanta or someone with ubuntu 20, can you try locally? (easiest might be to revert)

Reply:
Yeah its busted for me too, since its a mirror I wonder if the old commit will just work, worth a shot